device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/https/HttpsBatchMessage.java

```diff
@@ -170,7 +164,7 @@ private static String msgToJson(HttpsSingleMessage msg)
             for (String key : allProperties.keySet())
             {
                 jsonMsg.append("\"").append(key).append("\":");
-                jsonMsg.append("\"").append(allProperties.get(key)).append("\",");
+                jsonMsg.append("\"").append(allProperties.get(key)).append("\","); //TODO
```

Comment:
todo what?

Reply:
I'll send a PR out for this later. I've removed this todo for now